com.civicrm.hrjobroles/CRM/Hrjobroles/Upgrader.php

```diff
@@ -36,9 +36,11 @@ public function upgrade_1002(){
     return TRUE;
   }
```

Comment:
Any specific reason to split 1004 into two methods? I like having all that into a single method because it's easy to spot that we're updating the end_date because the type has changed